### PR TITLE
Added test for partial reload

### PR DESCRIPTION
**Pull request by Juhlinus**

### CODE REVIEW

File: tests/ResponseTest.php [hunk 1]
```diff
@@ -160,4 +160,27 @@ public function toArray()
...
+        $user = (object) ['name' => 'Jonathan'];
+        $response = new Response('User/Edit', ['user' => $user, 'partial' => 'partial-data'], 'app', '123');
+        $response = $response->toResponse($request);
+        $page = $response->getData();
+
+        $props = get_object_vars($page->props);
+
+        $this->assertInstanceOf(JsonResponse::class, $response);
+        $this->assertSame('User/Edit', $page->component);
+        $this->assertFalse(isset($props['name']));
```
Comment (victorlap) [inline tests/ResponseTest.php:180]: Shouldn’t you check for the user key here?

Comment (Juhlinus) [reply tests/ResponseTest.php:180]: The user key is not part of the partial data, the only key present is the `partial` key. See Line 169

Comment (victorlap) [reply tests/ResponseTest.php:180]: Yeah, that's why you want to check it is not there right? The `name` prop is within the `user` right?

```diff
- $this->assertFalse(isset($props['name']));
+ $this->assertFalse(isset($props['user']));

Comment (Juhlinus) [reply tests/ResponseTest.php:180]: Ah! I see what you mean now. Good catch!

I'll update this after work today. 😃

**Comment by reinink** [comment]
Looks good @Juhlinus with that one change. 💪

**Comment by Juhlinus** [comment]
@reinink done!

**Comment by reinink** [comment]
Thanks buddy! 💪

**Pull request merged**